pkg/sql/syntheticprivilege/external_connection_privilege.go

```diff
@@ -35,17 +33,19 @@ func (e *ExternalConnectionPrivilege) GetPath() string {
 	return fmt.Sprintf("/externalconn/%s", e.ConnectionName)
 }
 
-// GetPrivilegeDescriptor implements the PrivilegeObject interface.
-func (e *ExternalConnectionPrivilege) GetPrivilegeDescriptor(
-	ctx context.Context, planner eval.Planner,
-) (*catpb.PrivilegeDescriptor, error) {
-	if planner.IsActive(ctx, clusterversion.V22_2SystemPrivilegesTable) {
-		return planner.SynthesizePrivilegeDescriptor(ctx, e.GetPath(),
```

Comment:
I don't get it, what now looks up the privileges?

Reply:
It's still the planner, just without the indirection via the `eval.Planner` interface. See `grant_revoke_system.go`.